### ISSUE TITLE
Hansard error - IND MP Chaney marked 'absent' in House division 7th Mar 2023, 4:26 PM

### ISSUE THREAD

**Issue by mackaymackay** [openaustralia/theyvoteforyou#1463]
We've received an email from the Office of Kate Chaney MP alerting us to an error in Hansard.

Currently for a [House division on 7 March 2023](https://theyvoteforyou.org.au/divisions/representatives/2023-03-07/1), Ms Chaney is showing up as 'absent' when she actually was there and voted for the 'ayes'. The APH has updated this in their [Votes and Proceedings pdf](https://parlinfo.aph.gov.au/parlInfo/download/chamber/votes/aa7e859d-715d-4bde-b96b-79490288dce5/toc_pdf/reps-vp.pdf;fileType=application%2Fpdf#search=%22chamber/votes/aa7e859d-715d-4bde-b96b-79490288dce5/0000%22), however they haven't yet made the change to their [Hansard website](https://www.aph.gov.au/Parliamentary_Business/Hansard/Hansard_Display?bid=chamber/hansardr/26426/&sid=0000) or [Hansard pdf](https://parlinfo.aph.gov.au/parlInfo/download/chamber/hansardr/26426/toc_pdf/House%20of%20Representatives_2023_03_07.pdf;fileType=application%2Fpdf). 

I've let the person who alerted us know about the official Hansard not being updated yet, and hopefully the APH will sort this out quickly and we can fix it on our site too.

**Comment by katska** [comment]
@mackay Is this issue referring to[ the second reading division ](https://parlinfo.aph.gov.au/parlInfo/search/display/display.w3p;adv=yΩes;db=CHAMBER;id=chamber%2Fhansardr%2F26426%2F0134;orderBy=_fragment_number,doc_date-rev;page=0;query=Dataset%3Ahansardr,hansardr80%20Date%3A7%2F3%2F2023;rec=0;resCount=Default
)?  Kate Chaney MP is still not recorded in the votes there or in the subsequent division on amendment as far as I can see.

**Comment by mackaymackay** [comment]
@katska yes, it looks like the Hansard *still* hasn't been amended. I've followed up with another email to Ms Chaney's Office to ask for an update.

**Comment by mackaymackay** [comment]
Hansard has now been corrected, so Ms Chaney is [showing with the Ayes](https://www.aph.gov.au/Parliamentary_Business/Hansard/Hansard_Display?bid=chamber/hansardr/26426/&sid=0000), bringing the division to a total of 88 Ayes and 52 Noes.

We'll need to update [our record](https://theyvoteforyou.org.au/divisions/representatives/2023-03-07/1), as we're showing Ms Chaney as absent, with a total of 87 Ayes and 52 Noes.

**Comment by katska** [comment]
@mlandauer could you take a look at updating this as soon as you get a chance?

**Comment by mlandauer** [comment]
Fixed by reloading that day. I had to patch a different error where Tracey Roberts was referrred in voting records as "Roberts, S. R." rather than "Roberts, T. G.". It's a weird mistake.
